### PR TITLE
Fall cleaning

### PR DESCRIPTION
**Pull request by scjody**
Some much needed (IMO) cleanup for Plotbot.

@bpostlethwaite Please review.

### CODE REVIEW

File: plotbot.sample.conf [hunk 1]
```diff
@@ -15,54 +15,17 @@
...
-    "oauth_base_url": "http://localhost:8080 [must-correspond-with-your Google API Project in the dev console]",
-    "client_id": "some-client-id-after-setting-an-api-project-in-google",
-    "client_secret": "secret_keyfromAPIproject",
-    "restrict_domain": "hosted-domain.example.com",
-    "session_auth_key": "12312312312312312312312312312312",
-    "session_encrypt_key": "12345678902345678901234567890123"
-  },
-
-  "github": {
-    "authtoken": "put your github auth token here"
```
Comment (bpostlethwaite) [inline plotbot.sample.conf]: Isn't the authtoken still necessary to make github API calls? I know bug reporter plugin uses the github module.

**Comment by bpostlethwaite** [comment]
Could you also make a matching PR that removes the relevant sections from the [deployment plotbot config](https://github.com/plotly/deployment/blob/master/roles/plotbot/templates/plotbot.conf)?

**Comment by bpostlethwaite** [comment]
This is great! I agree much needed. Just a couple comments to consider.

File: plotbot.sample.conf (hunk 1 above)
Comment (scjody) [reply plotbot.sample.conf]: ✅

**Comment by scjody** [comment]
> Could you also make a matching PR

Done: https://github.com/plotly/deployment/pull/401

**Comment by bpostlethwaite** [comment]
Excellent! :dancer: 

A few more PRs like this and we might actually convince other people to jump in and help out ;)

**Pull request merged**